### PR TITLE
feat(tmp): Add openshift-redhat-marketplace in nss CR.

### PR DESCRIPTION
**Pull request by horis233**
We can remove it after we can input the namespace from OperandRegistry into the nss CR.

### CODE REVIEW

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Comment by pgodowski** [comment]
thanks. do we want also to add `kube-system` same way?

File: controllers/constant/namespace.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ metadata:
 spec:
   namespaceMembers:
   - placeholder
+  - openshift-redhat-marketplace
```
Comment (pgodowski) [inline controllers/constant/namespace.go:29]: do we want to add `kube-system` same way?

**Comment by horis233** [comment]
@pgodowski 
We can add `kube-system` in 1Q and remove them after we can get namespaces from the OperandRegistry in the fixpack.

**Pull request merged**